x-pack/legacy/plugins/siem/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -125,7 +121,7 @@ export const DraggableWrapper = React.memo<Props>(
                     <ConditionalPortal
                       isDragging={snapshot.isDragging}
                       registerProvider={registerProvider}
-                      usePortal={snapshot.isDragging && usePortal}
```

Comment:
have we reached a tipping point where the portal (originally for maps) is always required?

Reply:
it's just a middle step towards moving to `renderClone` https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/reparenting.md#rolling-your-own-portal